src/Obtainer/ObtainTable.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\migration_tools\Obtainer;
+
+/**
+ * Class ObtainTables
+ *
+ * Contains logic for parsing tables.
+ */
+class ObtainTable extends ObtainArray {
+  /**
+   * Extract contents from a table into an array.
+   *
+   * @param string $selector
+   *   The css selector of the item to search for (the parent item)
+   * @param int $table_num
+   *   The value of n where the table is the nth table on the page. E.g., 2 for
+   *   the second table on a page.
+   * @param string $method
+   *   What to build array with QP->text() or QP->html().
+   *
+   * @return array
+   *   The table array.
+   */
+  protected function extractTable($selector, $table_num, $method = 'text') {
```

Comment:
I'm concerned about muddying the water with 'extract'.    Extract seems to essentially be a finder as opposed to a plucker.
I  think was also want to preface all array finder and pluckers with 'array' like this one https://github.com/swirtSJW/migration_tools/blob/8.x-2.x/src/Obtainer/ObtainArray.php#L94

So I think we might want to consider renaming this to `arrayFindTable()`